cloud_governance/policy/common_policies/send_aggregated_alerts.py

```diff
@@ -141,7 +141,7 @@ def __update_delete_days(self, policy_es_data: list):
                     days = int(days)
                 if not days:
                     days = 0
-                alert_user = False
+                alert_user = True if self.__alert_dry_run else False
```

Comment:
alert_user should be False by default, can we do somthing like this ?
`alert_user = False if not self.__alert_dry_run else True`

Reply:
Both are same